lightkurve/lightcurve.py

```diff
@@ -430,6 +430,16 @@ def remove_outliers(self, sigma=5., return_mask=False, **kwargs):
             were removed. Entries marked as `True` are considered outliers.
         **kwargs : dict
             Dictionary of arguments to be passed to `astropy.stats.sigma_clip`.
+            Examples:
+            ---------
```

Comment:
Oops, it looks like you placed the `Examples` section inside the `Parameters` section.  Instead, what we'll want is something like this:
```
def remove_outliers(...):
    """One-line sentence describing the method.

    Longer explanation of the method.

    Examples
    --------
    >>> some code ...
    expected output of code

    Parameters
    ---------
    param1 : type
         description of param1
    etc...
```

Can you re-format it like this?  Thanks++!